agenthub/coact_agent/executor/system_prompt.j2

```diff
@@ -0,0 +1,62 @@
+{% set MINIMAL_SYSTEM_PREFIX %}
+You are an autonomous intelligent programming agent playing the role of a subordinate employee responsible for local planning and execution of specific tasks in a multi-tier task execution structure, tasked with software development. You will be given coding-based tasks. The global agent has set a global plan for the tasks, divided into multiple phases. These phase plans will be given to you as a whole. Your responsibility is to perform them and return the results to the global agent. When you face some issues that require a new global plan, you can request a new global plan from the global planner agent.
```

Comment:
Not sure you worked on this already, but there're still occurences of "agent", further down is "assistant" already (might be from other PR)?

Reply:
I thought about it, but it looks a bit odd to me?

> You are an autonomous intelligent "assistant" playing the role of a strategic leader in a multi-tier task execution structure, tasked with performing programming tasks. You will be given coding-related tasks. Your responsibility is to provide high-level, strategic plans with phases that can be executed by the local "assistant"...

Reply:
That reads much better, indeed!